### ISSUE TITLE
Learning rate and batch size adjustments

### ISSUE THREAD

**Issue by xiaoxi-private** [xiaoxi-s/transformer-with-shakespeare#7]
Adjust the following hyperparameters:
- `learning-rate`: 1e-5
- `batch-size`: 1024
